Neos.Flow/Tests/Unit/Mvc/Routing/Dto/UriConstraintsTest.php

```diff
@@ -86,6 +86,7 @@ public function applyToDataProvider()
             ['constraints' => [UriConstraints::CONSTRAINT_PATH_SUFFIX => 'suffix'], 'templateUri' => 'http://some-domain.tld', 'forceAbsoluteUri' => true, 'expectedUri' => 'http://some-domain.tld/suffix'],
             ['constraints' => [UriConstraints::CONSTRAINT_PATH_SUFFIX => 'suffix', UriConstraints::CONSTRAINT_PATH => '/some/path'], 'templateUri' => 'http://some-domain.tld', 'forceAbsoluteUri' => false, 'expectedUri' => '/some/pathsuffix'],
             ['constraints' => [UriConstraints::CONSTRAINT_PATH_SUFFIX => 'suffix', UriConstraints::CONSTRAINT_PATH => '/some/path'], 'templateUri' => 'http://some-domain.tld', 'forceAbsoluteUri' => true, 'expectedUri' => 'http://some-domain.tld/some/pathsuffix'],
+            ['constraints' => [UriConstraints::CONSTRAINT_PATH_SUFFIX => 'suffix', UriConstraints::CONSTRAINT_PATH_PREFIX => '/prefix/', UriConstraints::CONSTRAINT_PATH => '/some/path', UriConstraints::CONSTRAINT_HOST => 'localhost'], 'templateUri' => 'http://localhost/prefix/', 'forceAbsoluteUri' => false, 'expectedUri' => '/prefix/somepathsuffix'],
```

Comment:
```suggestion
            ['constraints' => [UriConstraints::CONSTRAINT_PATH_SUFFIX => 'suffix', UriConstraints::CONSTRAINT_PATH_PREFIX => '/prefix/', UriConstraints::CONSTRAINT_PATH => '/some/path', UriConstraints::CONSTRAINT_HOST => 'localhost'], 'templateUri' => 'http://localhost/prefix/', 'forceAbsoluteUri' => false, 'expectedUri' => '/prefix/some/pathsuffix'],
```
Shouldn't that be `/prefix/some/pathsuffx`?